### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by sammou00** [HYF-Class19/agile-development-loruki-group-2#20] [General]
Review

**Comment by sammou00** [comment]
Hello, well done. 
Here are some notes :

1- group members are working is not external constrains 
![image](https://user-images.githubusercontent.com/67711948/182945244-8b664128-44d1-4ee2-a822-9fddf489c284.png)

2- user-personas are always plurals not singular 
![image](https://user-images.githubusercontent.com/67711948/182945448-5a6d1898-6964-4ff5-a4c2-383101833e3d.png)

3- there is a little bite difference between your `backlog` and `strategy-plan` 
![image](https://user-images.githubusercontent.com/67711948/182945922-5a30a76d-5143-4413-b6c5-626e4349872f.png)
![image](https://user-images.githubusercontent.com/67711948/182946020-ec0ade4f-080c-4db3-9437-27d2639d408d.png)


4- nothing yet, in your `index.html` and `style.css`
![image](https://user-images.githubusercontent.com/67711948/182946183-1aeafb26-88cb-4187-aee0-57900ae8cfba.png)

**Reply by JohnOlutubo** [reply]
Thank you. All corrections well recieved. As of now, I have been trying to push the updated html and CSS files to github, but it fails to pass lint and validate. I hope to fix this with help of any group member.
